ontology/yaml/resources/METERS/entity_types/WM.yaml

```diff
@@ -35,4 +35,3 @@ WM_WF:
   opt_uses:
   - supply_water_volume_accumulator
   - water_linearvelocity_sensor
```

Comment:
Please sync this file, you are accidentally deleting a field from this type

Reply:
oops sorry, i thought `water_volume_accumulator` is the same as `supply_water_volume_accumulator` i thought I had added this a few months back and is not needed anymore.